.pylintrc

```diff
@@ -0,0 +1,3 @@
+[FORMAT]
+# PEP8 compliant
+max-line-length=79
```

Comment:
Ah, so PEP8 is stricter than Google Python Style Guide. Alright. However, we'll need to then fork the guidelines and adjust the column number in there. We should probably put the guidelines into https://github.com/RH-FMK/meta

Reply:
Maybe we can mention that we follow the Google Python Style Guide but pylint is on top of it. If you get a pylint error it is something that you have to fix. With the idea in mind to maintain less dev doc :)

Reply:
Yeah, we can have a "contributing" document refer to the style guide and have exceptions on top. That will work as long as exceptions are not numerous. We'll have to spell them out, though, not just point at the CI code.

Reply:
Ok, agreed